docs/using.rst

```diff
@@ -1,45 +1,109 @@
-==============================
-Using the Let's Encrypt client
-==============================
+==========
+User Guide
+==========
 
+.. contents:: Table of Contents
+   :local:
 
-Getting the code
-================
+.. _installation:
 
-Please `install Git`_ and run the following commands:
+Installation
+============
+
+Unless you have a very specific requirements, we kindly ask you to use
+the letsencrypt-auto_ method described below. It's the fastest, the
```

Comment:
Please remove this paragraph.  Rather than opening with a discussion of different installation methods, we should open with simple instructions for the best one.  If you really want to put this in the help, put it in the section where we talk about other ways of installing / running the client.